app/controllers/idv/by_mail/enter_code_controller.rb

```diff
@@ -19,7 +19,7 @@ def index
         )
 
         if rate_limiter.limited?
-          render_rate_limited
+          redirect_to idv_enter_code_rate_limited_path
```

Comment:
We want to redirect to urls, not paths.
```suggestion
          redirect_to idv_enter_code_rate_limited_url
```

Reply:
>We want to redirect to urls, not paths.

Is this an IdV-specific thing, or a general rule of thumb? Personally I've always preferred the `_path` variants where possible to avoid dealing with risks and verbosity of hostnames.

Reply:
My understanding is that it introduces a vulnerability to redirect to a path rather than a full url. I got a correction on a PR recently but I'm not finding it right now.

Reply:
As a quick clarification, there shouldn't be a significant difference with regards to open redirect vulnerabilities.

Rails will ultimately convert [a path to a URL](https://github.com/rails/rails/blob/04972d9b9ef60796dc8f0917817b5392d61fcf09/actionpack/lib/action_controller/metal/redirecting.rb#L137-L140) based on the request, and will always do a check to [enforce open redirect protection](https://github.com/rails/rails/blob/04972d9b9ef60796dc8f0917817b5392d61fcf09/actionpack/lib/action_controller/metal/redirecting.rb#L89) unless the `allow_other_host` option is enabled (like we do for [OIDC redirects](https://github.com/18F/identity-idp/blob/398c1a53e4e748c8f95e9a8671bf2263c95f6c59/app/controllers/openid_connect/authorization_controller.rb#L76))

Reply:
Thanks for the update, @mitchellhenke.